cpp/zeder_to_zotero_importer.cc

```diff
@@ -270,13 +271,28 @@ unsigned ImportZederEntries(const Zeder::EntryCollection &zeder_entries, Harvest
 
         auto existing_journal_section(harvester_config->lookupConfig(zeder_id, zeder_flavour));
         if (existing_journal_section != nullptr and not overwrite) {
-            LOG_WARNING("couldn't import Zeder entry " + std::to_string(zeder_id) + " (" + title + "): already exists");
+            if (not autodetect_new_datasets) {
+                LOG_WARNING("couldn't import Zeder entry " + std::to_string(zeder_id) + " (" + title + "): already exists");
+                continue;
+            } else {
+                LOG_INFO("Skipping Zeder entry " + std::to_string(zeder_id) + " (" + title + "): already exists");
+                continue;
+            }
```

Comment:
As you continue in both cases, how about factoring that out and putting it below the else-case?  I'd also switch the if- and else- clauses because then you van drop the "not".

Reply:
done!